{"name": "test_sub", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_pytest1.Test_pytest object at 0x0000014F89A88F50>\n\n    def test_sub(self):\n        a=10\n        b=20\n        print(a-b)\n        if a==b:\n            print(\"it is true\")\n            assert True\n        else:\n            print(\"it is false\")\n>           assert False\nE           assert False\n\ntest_pytest1.py:15: AssertionError"}, "start": 1690647028279, "stop": 1690647028284, "uuid": "021a8ee6-eb8c-4a8f-ba7a-2a00fa546388", "historyId": "a8b02618e153e89f8784a829b91878d6", "testCaseId": "a8b02618e153e89f8784a829b91878d6", "fullName": "test_pytest1.Test_pytest#test_sub", "labels": [{"name": "suite", "value": "test_pytest1"}, {"name": "subSuite", "value": "Test_pytest"}, {"name": "host", "value": "DESKTOP-0UK8E2L"}, {"name": "thread", "value": "7248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest1"}]}